app/internal/page/app.go

```diff
@@ -346,6 +347,7 @@ func makeHandle(mux *http.ServeMux, logger Logger, store sessions.Store, localiz
 				CookieConsentSet: cookieErr != http.ErrNoCookie,
 				CanGoBack:        opt&CanGoBack != 0,
 				RumConfig:        rumConfig,
+				PreviousPath:     r.Header.Get("Referer"),
```

Comment:
this just creates a loop between 2 pages, maybe it should be done in JS so it uses the browser history (because I would hate us to have to store a list server side to play back) _or_ I now think we should remove the back button completely because it will be very annoying/difficult to get it to work in a logical way, and we can come back to it later if/when we have more research supporting us adding one

Reply:
😂 I knew there was going to be some flaw in this. I'd only tested going back a single page, making a change, and continuing.

I'll see what the appetite is for dropping this for UR in standup tomorrow - I agree other options are going to be complex for what seems like replicating the browser back button.